models/points.go

```diff
@@ -1193,6 +1195,7 @@ func scanFieldValue(buf []byte, i int) (int, []byte) {
 	return i, buf[start:i]
 }
 
+// EscapeMeasurement measures escape codes
```

Comment:
`// EscapeMeasurement escapes all special characters in the provided measurement.`